code/cubical/Term/Category.agda

```diff
@@ -0,0 +1,61 @@
+{- Monoidal Category of SetGrammars and Terms for a fixed hLevel -}
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+
+module Term.Category (Alphabet : hSet ℓ-zero) where
+
+open import Cubical.Categories.Category.Base
+open import Cubical.Categories.Functor
+open import Cubical.Categories.NaturalTransformation
+open import Cubical.Categories.Monoidal.Base
+
+open import Grammar.Base Alphabet
+open import Grammar.HLevels Alphabet
+open import Grammar.Epsilon Alphabet
+open import Grammar.LinearProduct Alphabet as LP
+open import Term.Base Alphabet as Term
+
+module _ (ℓ : Level) where
+  open MonoidalCategory
+  open TensorStr
+  open MonoidalStr
+  open Category
+  open Functor
+  open NatTrans
+  open NatIso
+  open isIso
+  |GRAMMAR| : Category (ℓ-suc ℓ) ℓ
```

Comment:
We've talked about how it would be less universe polymorphic to do so, but defining this as a category could now let us replace `unambiguous` and `StrongEquivalence` with the definition of mono and iso from the categories library